Experiments/Experiments.xcodeproj/project.pbxproj

```diff
@@ -13,9 +13,9 @@
 		0270C0A327069B7800FC799F /* FeatureFlagService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0270C0A227069B7800FC799F /* FeatureFlagService.swift */; };
 		0270C0A527069B8900FC799F /* DefaultFeatureFlagService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0270C0A427069B8900FC799F /* DefaultFeatureFlagService.swift */; };
 		0270C0A727069BA500FC799F /* BuildConfiguration.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0270C0A627069BA500FC799F /* BuildConfiguration.swift */; };
-		3F8115032DD2A6EC0042DDB6 /* AutomatticTracks in Frameworks */ = {isa = PBXBuildFile; productRef = 3F8115022DD2A6EC0042DDB6 /* AutomatticTracks */; };
+		3F2B4ACA2DDC30E000E5E49C /* XcodeTarget_Experiments in Frameworks */ = {isa = PBXBuildFile; productRef = 3F2B4AC92DDC30E000E5E49C /* XcodeTarget_Experiments */; };
+		3F2B4ACC2DDC30E800E5E49C /* XcodeTarget_ExperimentsTests in Frameworks */ = {isa = PBXBuildFile; productRef = 3F2B4ACB2DDC30E800E5E49C /* XcodeTarget_ExperimentsTests */; };
 		3F8B84192DD2FEC2002DD6E8 /* Experiments.xctestplan in Resources */ = {isa = PBXBuildFile; fileRef = 3F8B84182DD2FEC2002DD6E8 /* Experiments.xctestplan */; };
-		3F8B841B2DD301C6002DD6E8 /* AutomatticTracks in Frameworks */ = {isa = PBXBuildFile; productRef = 3F8B841A2DD301C6002DD6E8 /* AutomatticTracks */; };
```

Comment:
As per the PR description, the Experiments and ExperimentsTests targets (frameworks) now have dedicated `XcodeTarget_Experiments` and `XcodeTarget_ExperimentsTests` packages where all their dependencies are defined.

As such, you can see here that the `AutomatticTracks` links are gone, replaced`XcodeTarget_Experiments[Tests]`.

Notice that as soon as Experiments and its tests become modules themselves, all this will be greatly simplified as a standard package description. 

The only  `XcodeTarget_` supporting targets that will remain are the ones for the app targets.

Reply:
That's correct @staskus 

> you can move everything to the Package in one sweep rather than only when the framework subproject is turned into a package.

That's the tactical advantage of introducing the change now...

> Also, now the dependencies of each framework are defined within the xcodeTarget.

...and that's the long term benefit. Once all the frameworks will be packages, we'll have `XcodeTarget_<name>` only for the app targets, the notification extensions, and the test targets. `Modules/Package.swift` will be the single source of truth for dependencies, whether first- or third-party. And using a Swift file is much better to modify and understand than the Xcode GUI.